packages/php-db-import-export/tests/unit/Storage/GCS/ManifestGenerator/GCSSlicedManifestFromFolderGeneratorTest.php

```diff
@@ -40,23 +47,10 @@ public function testGenerateAndSaveManifest(): void
         $bucket->expects($this->exactly(2))
             ->method('name')
             ->willReturn('bucket1');
-        $bucket->expects($this->once())
-            ->method('upload')
-            ->with(
-                $this->callback(function ($subject) {
-                    $this->assertIsResource($subject);
-                    $content = stream_get_contents($subject);
-                    $this->assertSame(
-                        //phpcs:ignore
-                        '{"entries":[{"url":"gs:\\/\\/bucket1\\/prefix\\/xxx\\/obj1_000000.csv","mandatory":true},{"url":"gs:\\/\\/bucket1\\/prefix\\/xxx\\/obj2_000000.csv","mandatory":true}]}',
-                        $content,
-                    );
-                    return true;
-                }),
-                [
-                    'name' => 'prefix/xxxmanifest',
-                ],
-            );
+
+        $bucket->expects($this->once())->method('getStreamableUploader')
+            ->willReturn($streamableUploaderMock);
+
```

Comment:
Přemýšlám jestli ten test k něčemu je když sa neotestuje to že to posílá :thinking:

Reply:
Teda ne že by to teď testoval když sa upload nevolá :D

Reply:
Taky sem to evidentně moc dobře nenapsal už podle názvu metody z toho čiší single-responsibility princip :D